api/models/Community.js

```diff
@@ -92,28 +93,22 @@ module.exports = bookshelf.Model.extend(merge({
 
   createStarterPosts: function (transacting) {
     var now = new Date()
-    var timeShift = {null: 0, intention: 1, offer: 2, request: 3}
-    return Community.find('starter-posts', {withRelated: [
-      'posts', 'posts.followers', 'posts.selectedTags'
-    ]})
+    var timeShift = {offer: 1, request: 2}
```

Comment:
Looks like we only have 1 starter post in new communities now, so I think we can remove this timeshift stuff. And not in scope for this work so no worries if you don't